### PR TITLE
Template 2.3 update 

### PR DESCRIPTION
**Pull request by ypriverol**
## PR checklist

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
    - [x] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/quantms/tree/master/.github/CONTRIBUTING.md)
    - [x] If necessary, also make a PR on the nf-core/quantms _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [x] Usage Documentation in `docs/usage.md` is updated.
- [x] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [x] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Comment by ypriverol** [comment]
@jpfeuffer still working on this, do not merge.

**Comment by jpfeuffer** [comment]
linting still fails. Dont you want to ignore it?

**Comment by jpfeuffer** [comment]
and yaml linting has hundreds of errors

File: README.md [hunk 1]
```diff
@@ -40,22 +40,22 @@ On release, automated continuous integration tests run the pipeline on a full-si
...
+    > * If you are using `singularity`, please use the [`nf-core download`](https://nf-co.re/tools/#downloading-pipelines-for-offline-use) command to download images first, before running the pipeline. Setting the [`NXF_SINGULARITY_CACHEDIR` or `singularity.cacheDir`](https://www.nextflow.io/docs/latest/singularity.html?#singularity-docker-hub) Nextflow options enables you to store and re-use the images from a central location for future pipeline runs.
     > * If you are using `conda`, it is highly recommended to use the [`NXF_CONDA_CACHEDIR` or `conda.cacheDir`](https://www.nextflow.io/docs/latest/conda.html) settings to store the environments in a central location for future pipeline runs.
 
 4. Start running your own analysis!
 
     <!-- TODO nf-core: Update the example "typical command" below used to run the pipeline -->
 
     ```console
-    nextflow run nf-core/quantms -profile <docker/singularity/podman/shifter/charliecloud/conda/institute> --input samplesheet.csv --genome GRCh37
+    nextflow run nf-core/quantms --input samplesheet.csv --outdir <OUTDIR> --genome GRCh37 -profile <docker/singularity/podman/shifter/charliecloud/conda/institute>
```
Comment (jpfeuffer) [inline README.md:58]: we dont have csv and we do not have genome

File: docs/usage.md [hunk 2]
```diff
@@ -6,21 +6,59 @@
 
 ## Introduction
 
-## Running the pipeline
+<!-- TODO nf-core: Add documentation about anything specific to running your pipeline. For general topics, please point to (and add to) the main nf-core website. -->
```
Comment (jpfeuffer) [inline docs/usage.md:9]: remove todo

File: docs/usage.md [hunk 3]
```diff
@@ -6,21 +6,59 @@
...
+```console
+--input '[path to samplesheet file]'
+```
+
+### Multiple runs of the same sample
+
+The `sample` identifiers have to be the same when you have re-sequenced the same sample more than once e.g. to increase sequencing depth. The pipeline will concatenate the raw reads before performing any downstream analysis. Below is an example for the same sample sequenced across 3 lanes:
+
+```console
+sample,fastq_1,fastq_2
```
Comment (jpfeuffer) [inline docs/usage.md:24]: please adapt or remove

File: docs/usage.md [hunk 4]
```diff
@@ -6,21 +6,59 @@
...
+An [example samplesheet](../assets/samplesheet.csv) has been provided with the pipeline.
 
-In the respective "comment[file uri]" or "Spectra_Filepath" columns, the raw or mzML files with the mass spectra to be staged have to be listed. URIs are possible,
-and the root folder as well as the file endings can be changed in the options in case of previously downloaded, moved or converted experiments.
+## Running the pipeline
+
+The typical command for running the pipeline is as follows:
+
+```console
+nextflow run nf-core/quantms --input samplesheet.csv  --outdir <OUTDIR> --genome GRCh37 -profile docker
```
Comment (jpfeuffer) [inline docs/usage.md:60]: please adapt or remove.
Just leave the old text

File: lib/Utils.groovy [hunk 5]
```diff
@@ -29,19 +29,12 @@ class Utils {
...
                 "  NB: The order of the channels matters!\n" +
-                "==================================================================================="
+                "~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~"
         }
     }
-
-    //
-    // Join module args with appropriate spacing
-    //
-    public static String joinModuleArgs(args_list) {
```
Comment (jpfeuffer) [inline lib/Utils.groovy:44]: we might need this. wait for actual CI

**Comment by ypriverol** [comment]
@jpfeuffer what is pending ?

**Comment by jpfeuffer** [comment]
You resolved unresolved comments

File: docs/usage.md (hunk 3 above)
Comment (ypriverol) [reply docs/usage.md]: Should we write in this PR the usage complelty?

File: docs/usage.md (hunk 4 above)
Comment (ypriverol) [reply docs/usage.md]: I change that, this is the version now: 

```bash
nextflow run nf-core/quantms --input '/url/path/to/your/experimentX_design.tsv' --database '/url/path/to/your/proteindatabase.fasta' -profile docker
```

**Comment by jpfeuffer** [comment]
They show up under my second review.

**Comment by jpfeuffer** [comment]
weird. it still shows in the Files Changed view.

**Pull request merged**